<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="pose_x" default="0.0"/>
	<arg name="pose_y" default="0.0"/>
	<arg name="pose_z" default="0.0"/>
	<arg name="pose_yaw" default="0.0"/>
	<arg name="robot_id" default="0"/>

	<arg name="run_cond" default="1"/>

	<group ns="satlet0">
		<include file="$(find abp_sim)/launch/satlet.launch">
			<arg name="pose_x" value="0"/>
			<arg name="pose_y" value="$(arg pose_y)"/>
			<arg name="pose_z" value="$(arg pose_z)"/>
			<arg name="pose_yaw" value="$(arg pose_yaw)"/>
			<arg name="robot_id" value="0"/>
			<arg name="run_cond" default="1"/>

		</include>
	</group>
	<group ns="satlet1">
		<include file="$(find abp_sim)/launch/satlet.launch">
			<arg name="pose_x" value="-2"/>
			<arg name="pose_y" value="$(arg pose_y)"/>
			<arg name="pose_z" value="$(arg pose_z)"/>
			<arg name="pose_yaw" value="$(arg pose_yaw)"/>
			<arg name="robot_id" value="1"/>
			<arg name="run_cond" default="1"/>

		</include>
	</group>
</launch>
